src/test/java/tests/BaseTest.java

```diff
@@ -1,8 +1,50 @@
 package tests;
 
-import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.core.AbstractTest;
+import com.zebrunner.carina.webdriver.IDriverPool;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.AfterMethod;
 
-public abstract class BaseTest implements IAbstractTest {
+public abstract class BaseTest extends AbstractTest implements IDriverPool {
 
-}
+    protected WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
```

Comment:
Use @Parameters instead of System.getProperty for driver_type, and helper methods setupAppiumDriver and   setupSeleniumDriver
@Parameters("driver_type")
public void setUp(String driverType) {
    if ("APPIUM".equalsIgnoreCase(driverType)) {
        setupAppiumDriver();
    } else {
        setupSeleniumDriver();
    }
}